atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyMessagingService.java

```diff
@@ -375,6 +378,15 @@ public CompletableFuture<MessagingService> start() {
         .thenCompose(ok -> bootstrapServer())
         .thenRun(
             () -> {
+              final var metrics = new NettyDnsMetrics();
+              dnsResolverGroup =
+                  new DnsAddressResolverGroup(
+                      new DnsNameResolverBuilder(clientGroup.next())
+                          .dnsQueryLifecycleObserverFactory(
+                              new BiDnsQueryLifecycleObserverFactory(
+                                  ignored -> metrics,
```

Comment:
🔧 With Java 21 we can now use `_` https://openjdk.org/jeps/443

Reply:
IntelliJ flags this as a preview feature :thinking: